react-client/.env.development

```
@@ -0,0 +1,2 @@
+VITE_API_BASE_URL=http://localhost:3000
+VITE_SHOW_DEMO_CREDS=true
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)